android-auto/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    package="com.mapbox.onetap.androidauto">
+
+    <uses-permission android:name="androidx.car.app.NAVIGATION_TEMPLATES"/>
+    <uses-permission android:name="androidx.car.app.ACCESS_SURFACE"/>
+
+    <application>
+
+        <service
+            android:name="com.mapbox.onetap.androidauto.car.MainCarAppService"
```

Comment:
```suggestion
            android:name="com.mapbox.androidauto.car.MainCarAppService"
```

Reply:
hmm this is going to be a painful change 🤔

Reply:
I know 😬

Reply:
Yep, let's do it! 🚀